docs/content/Getting-Started/user-quickstart.md

```diff
@@ -0,0 +1,253 @@
+        nodes:
+        - role: control-plane
+          kubeadmConfigPatches:
+          - |
+            kind: InitConfiguration
+            nodeRegistration:
+              kubeletExtraArgs:
+                node-labels: "ingress-ready=true"
+          extraPortMappings:
+          - containerPort: 443
+            hostPort: {{ config.ks_port_num }}
+            protocol: TCP
+        EOF
+        ```
+
+        Be sure to apply an ingress control with SSL passthrough to 'ks-core'. This is a special requirement for Kind that allows access to the KubeStellar core running on 'ks-core'.
+        ```
+        kubectl apply -f https://raw.githubusercontent.com/kubestellar/kubestellar/main/example/kind-nginx-ingress-with-SSL-passthrough.yaml
+        ```
+        **Wait about 10 seconds** and then check for ingress to be ready:
+        ```
+        kubectl wait --namespace ingress-nginx \
+          --for=condition=ready pod \
+          --selector=app.kubernetes.io/component=controller \
+          --timeout=90s
+        ```
+
+    <!-- === "edge-cluster1" -->
+        create the edge-cluster1 kind cluster
+        ```
+        kind create cluster --name edge-cluster1 --config=- <<EOF
+        kind: Cluster
+        apiVersion: kind.x-k8s.io/v1alpha4
+        nodes:
+        - role: control-plane
+          extraPortMappings:
+          - containerPort: 8081
+            hostPort: 8094
+        EOF
+        ```
+
+    <!-- === "edge-cluster2" -->
+        create the edge-cluster2 kind cluster
+        ```
+        kind create cluster --name edge-cluster2 --config=- <<EOF
+        kind: Cluster
+        apiVersion: kind.x-k8s.io/v1alpha4
+        nodes:
+        - role: control-plane
+          extraPortMappings:
+          - containerPort: 8081
+            hostPort: 8096
+          - containerPort: 8082
+            hostPort: 8097
+        EOF
+        ```
+   
+#### 1. Deploy your KubeStellar Core component
+!!! tip ""
+    === "deploy"
+         ```
+         # deploy KubeStellar core components on the 'ks-core' kind cluster you created in the pre-req section above
+         KUBECONFIG=~/.kube/config kubectl config use-context kind-ks-core
+         kubectl create namespace kubestellar
+         helm repo add kubestellar https://helm.kubestellar.io
+         helm install kubestellar/kubestellar-core --set EXTERNAL_HOSTNAME="localhost" --set EXTERNAL_PORT={{ config.ks_port_num }} --namespace kubestellar --generate-name
+         ```
+    === "when is KubeStellar ready?"
+         run the following to wait for KubeStellar to be ready to take requests:
+         ```
+         echo -n 'Waiting for KubeStellar to be ready'
+         while ! kubectl exec $(kubectl get pod --selector=app=kubestellar -o jsonpath='{.items[0].metadata.name}' -n kubestellar) -n kubestellar -c init -- ls /home/kubestellar/ready &> /dev/null; do
+            sleep 10
+            echo -n "."
+         done
+         echo "KubeStellar is now ready to take requests"
```

Comment:
One could add 1 or 2 newline(s) before echoing "KubeStellar is now ready to take requests".
This would avoid to echo a cluttered output message when the user is too slow to cut and paste the wait command, such as:
  --> "Waiting for KubeStellar to be readyKubeStellar is now ready to take requests"